__tests__/app.test.js

```diff
@@ -290,4 +290,37 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(response.body.msg).toBe('Invalid votes value. Must be an integer.');
       });
   });
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test('DELETE:204 deletes the specified comment, ensure that the comment is no longer in the databse', () => {
+    return request(app)
+    .delete('/api/comments/1')
+    .expect(204)
+    .then(() => {
+      return request(app)
+      .get('/api/comments/1')
+      .expect(404)
+    })
+    .then(({body}) => {
+      expect(body.msg).toBe("comment does not exist")
+    })
```

Comment:
Nice test! 🥇 Some people might say this is a little excessive, if we manage to get to a 2xx status then the server did it's thing and completed OK. You also then move into testing error handling within the 204 test! But it's good to see you're thinking ahead and others would say this is how it should be done 👍